api/lib/domain/constants.js

```diff
@@ -55,6 +55,7 @@ const LOCALE = {
   FRENCH_FRANCE: 'fr-fr',
   FRENCH_SPOKEN: 'fr',
 };
+const SUPPORTED_LOCALES = ['en', 'fr', 'en-GB', 'fr-BE', 'fr-FR'];
```

Comment:
```suggestion
const SUPPORTED_LOCALES = ['en', 'fr', 'fr-BE', 'fr-FR'];
```

Transformer la locale 'en-GB' en 'en'